business_requirement_deliverable/views/business_view.xml

```diff
@@ -31,14 +31,14 @@
             <field name="model">business.requirement.deliverable</field>
             <field name="arch" type="xml">
                 <tree string="Business Requirement Deliverable" default_order="sequence">
-                    <field name="currency_id" invisible="1"/>
+                    <field name="currency_id" invisible="1" groups="business_requirement_deliverable.group_business_requirement_estimation"/>
```

Comment:
@victormartinelicocorp 
What I want is https://github.com/OCA/business-requirement/pull/100#discussion_r108822624
I trust that you will find the right method to do it.